packages/@icon-magic/distribute/src/helpers/create-image-set.ts

```diff
@@ -0,0 +1,100 @@
+import { Logger, logger } from '@icon-magic/logger';
```

Comment:
None of these files are really helpers. I like think of helpers being more of utility/helper functions. We can have them directory under /src